### PR TITLE
LPS-80386 Merge UADAggregator into UADDisplay

### PR DESCRIPTION
**Pull request by pei-jung**
/cc @drewbrokke @willnewbury

### CODE REVIEW

**Comment by pei-jung** [comment]
ci:test

File: modules/util/portal-tools-service-builder/src/main/java/com/liferay/portal/tools/service/builder/ServiceBuilder.java [hunk 1]
```diff
@@ -6575,20 +6575,6 @@ private void _removeServletContextUtil(String outputPath) {
...
-				"UADAggregator.java"));
-	}
-
-	private void _removeUADAggregatorTest(Entity entity) {
-		_deleteFile(
-			StringBundler.concat(
-				_uadTestIntegrationOutputPath, "/uad/aggregator/test/",
-				entity.getName(), "UADAggregatorTest.java"));
-	}
-
```
Comment (drewbrokke) [inline modules/util/portal-tools-service-builder/src/main/java/com/liferay/portal/tools/service/builder/ServiceBuilder.java]: Thanks for catching this, @pei-jung

**Comment by brianchandotcom** [comment]
@pei-jung rebase error, can you resend? Thx.

**Comment by drewbrokke** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**